airbyte-cdk/java/airbyte-cdk/build.gradle

```diff
@@ -30,8 +30,8 @@ subprojects { subproject ->
         }
         repositories {
             maven {
-                name 'airbyte-public-jars'
-                url 'https://airbyte.mycloudrepo.io/repositories/airbyte-public-jars/'
+                name 'airbyte-repo'
+                url 'https://airbyte.mycloudrepo.io/repositories/airbyte-public-jars/' //This is only used to publish. 
```

Comment:
nit, regarding commentary: I had in mind also a comment which mentioned that this URL was different than that in the repositories { } block in settings.gradle.

Reply:
@postamar ack and changed